entries/q/qiwi.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Qiwi": {
+    "domain": "qiwi.com",
+    "contact": {
+      "email": "bankinfo@qiwi.com",
+      "twitter": "QiwiRussia"
+    },
+    "keywords": [
+      "payments"
```

Comment:
The payment category is meant for services like Apple Pay, Visa, Mastercard, Stripe etc.
Is Qiwi a payment service or a bank?

Reply:
It is a payment system similar to Yoomoney, Payeer and Skrill.